trade-finance-manager-ui/templates/case/amendments/amendment-cover-end-date.njk

```diff
@@ -20,59 +20,67 @@
       classes: "govuk-!-margin-top-4 govuk-!-margin-bottom-4"
     }) }}
   {% endif %}
-  <h1 class="govuk-heading-l govuk-!-margin-top-4 govuk-!-margin-bottom-2" data-cy="amendment--cover-end-date-heading" aria-label="Enter the new cover end date">Enter the new cover end date</h1>
 
-  <div class="govuk-body govuk-!-margin-bottom-4" data-cy="amendment--current-cover-end-date">
-    <p class="govuk-body govuk-!-margin-bottom-0 govuk-!-font-weight-bold">Current cover end date</p>
-    {{currentCoverEndDate}}
-  </div>
   <form method="POST" autocomplete="off">
     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-    {{ govukDateInput({
-      id: "coverEndDate",
-      namePrefix: "cover-end-date",
-      hint: {
-        text: "For example, 31 3 2022",
-        attributes: {
-          "data-cy": "cover-end-date-hint"
-        }
-      },
-      errorMessage: errors and errors.fieldErrors.coverEndDate and {
-        text: errors.fieldErrors.coverEndDate.text,
-        attributes: {
-          'data-cy': 'amendment--inline-error'
-        }
-      },
-      items: [
-        {
-          label: "Day",
-          classes: errors.fieldErrors.coverEndDate and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
-          name: "day",
-          value: coverEndDateDay,
+
+    <fieldset class="govuk-fieldset" role="group" aria-describedby="amendmentRequestDate-hint">
+
+      <legend class="govuk-fieldset__legend">
+        <h1 class="govuk-heading-l govuk-!-margin-top-4 govuk-!-margin-bottom-0" data-cy="amendment--cover-end-date-heading">Enter the new cover end date</h1>
+      </legend>
+
+      <div class="govuk-body govuk-!-margin-bottom-4" data-cy="amendment--current-cover-end-date">
+        <p class="govuk-body govuk-!-margin-bottom-0 govuk-!-font-weight-bold">Current cover end date</p>
+        {{currentCoverEndDate}}
```

Comment:
I know this wasn't mentioned in this ticket, but do you think we should change this to a description list (`<dl><dt><dd>`)?

Reply:
yes, it is single item, but dl might be good, changed.